PinpointKit/PinpointKit/Sources/Fonts.swift

```diff
@@ -32,7 +32,7 @@ public extension UIFont {
      
      - returns: A Source Sans Pro `UIFont` at the specified size and weight.
      */
-    public static func sourceSansProFontOfSize(_ fontSize: CGFloat, weight: FontWeight = .regular) -> UIFont {
+    public static func sourceSansProFont(ofSize fontSize: CGFloat, weight: FontWeight = .regular) -> UIFont {
```

Comment:
Which `font` do you propose removing? The one from `sourceSansProFont` or `fontSize`?

I was deliberately keeping these in line with ’s:

```swift
public class func systemFont(ofSize fontSize: CGFloat) -> UIFont
```

But I see how you could make an argument similar to `UIColor.whateverColor()` → `UIColor.whatever()`.